console/jsre/deps/doc.go

```diff
@@ -18,5 +18,18 @@
 // This file is derived from internal/jsre/deps/deps.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package deps contains the console JavaScript dependencies Go embedded.
+/*
+Package deps contains the console JavaScript dependencies Go embedded.
+
+Klaytn console provides functions of bignumber.js and web3.js which are included in the source tree. To make those available, deps is using go-bindata<https://github.com/jteeuwen/go-bindata> to make those javascript source files into assets which can be used by go environment.
+
+Source Files
+
+Each file provides following features
+  - deps.go	: Define `go:generate` options to use go-bindata in making bignumber.js and web3.js as assets
```

Comment:
```suggestion
  - deps.go	: Defines `go:generate` options to use go-bindata in making bignumber.js and web3.js as assets
```

Reply:
@aidan-kwon Thank you for pointing it out. Please take another look.